package/CHANGELOG

```diff
@@ -18,6 +18,8 @@ The rules for this file:
   * 2.0.0
 
 Fixes
+  * Selections on emtpy AtomGroups now return an empty AtomGroup instead of an
```

Comment:
Added this one in as requested by @lilyminium

Reply:
Thank you! Could you add in my username above as well?

Reply:
Oops my bad, done